LICENSE

```diff
@@ -1,7 +1,7 @@
 
                                  Apache License
                            Version 2.0, January 2004
-                        http://www.apache.org/licenses/
+                        https://www.apache.org/licenses/
```

Comment:
This change was a bit interesting. I had to see if it was OK and it appears it is -- https://opensource.stackexchange.com/questions/4200/is-it-ok-to-change-url-scheme-for-the-links-to-apache-2-license